init-tools.sh

```diff
@@ -132,7 +132,13 @@ if [ ! -e $__DOTNET_PATH ]; then
         cd $__DOTNET_PATH
         tar -xf $__DOTNET_PATH/dotnet.tar
     }
-    execute_with_retry install_dotnet_cli >> "$__init_tools_log" 2>&1
+
+    if [[ "$DotNetBootstrapCliTarPath" = "" ]]; then
+        __execute_with_retry install_dotnet_cli >> "$__init_tools_log" 2>&1
```

Comment:
Shouldn't this just be `execute_with_retry`?

Reply:
Yes.  typo.

Reply:
Amazing.  Even with you pointing it out it took a minute to see it.

Done